README.md

```diff
@@ -387,6 +389,15 @@ removed thislimitation.
   can also be configured
   Default is `[]`.
 
+#### Secrets Configuration
+
+- **[`secrets`](#secrets-object-attributes)**: _(Optional `list(secret)`)_
+
+  This resource allows you to create and manage secrets for repositories in your organization.
+  When applied, a secret with the given key and value will be created in the repositories.
+  The value of the secrets must be given in plain text, github provider is in charge of encrypting it.
```

Comment:
May be good to add a Note about not clicking those into your github.

e.g.

```suggestion
  The value of the secrets must be given in plain text, github provider is in charge of encrypting it.
  **Attention:** You might want to get secrets via a data source from a secure vault and not add them in plain text to your source files; so you do not check plaintext secrets into the git repository managing your github account.
```

Reply:
Better, thanks for the suggestion!